cypress/utils/utils.ts

```diff
@@ -1858,7 +1858,7 @@ export function validateTackleCr(): void {
 
 export function validateMtaOperatorLog(): void {
     cy.wait(30 * SEC);
-    let command = `oc logs $(oc get pods | grep mta-operator | cut -d " " -f 1) | grep failed | tail -n 1| awk -F 'failed=' '{print $2}'|cut -d " " -f 1`;
+    let command = `oc logs $(oc get pods -n openshift-mta | grep mta-operator | cut -d " " -f 1) -n openshift-mta | grep failed | tail -n 1| awk -F 'failed=' '{print $2}'|cut -d " " -f 1`;
     getCommandOutput(command).then((result) => {
         const failedCount = parseInt(result.stdout.trim());
         if (Number.isNaN(failedCount)) {
```

Comment:
Lines 1864-1868 can be removed. I added them for debugging

Reply:
Done